### ISSUE TITLE
Extended networking tests fail on "Path /var/run/openvswitch/br0.mgmt is mounted on /run but it is not a shared or slave mount."

### ISSUE THREAD

**Issue by dcbw** [openshift/origin#11707]
```
Oct 31 11:16:25.304: INFO: At 2016-10-31 11:15:44 -0400 EDT - event for flow-checkl35gu: {kubelet nettest-node-1} FailedSync: Error syncing pod, skipping: failed to "StartContainer" for "flow-check" with RunContainerError: "runContainer: Error response from daemon: Cannot start container ec598cdedcc673560b770debe1b1d705e3fc0751182dcc82b0ba5c740ec846f1: Path /var/run/openvswitch/br0.mgmt is mounted on /run but it is not a shared or slave mount."
```

**Comment by marun** [comment]
I'm seeing this in the post-merge job, which suggests this is a problem in master rather than just #11613.  I'm puzzled what changed to cause this apparent regression.

**Comment by liggitt** [comment]
Did docker just change to 1.12?

**Comment by marun** [comment]
Hmm, that may be it.  That is the version shown on the failing jobs.

**Comment by marun** [comment]
To be clear, this is only a problem for dind clusters and the fix is to ensure /run is mounted with shared propagation as per the fix included with @dcbw's PR.  Only the ovs e2e tests are affected, and they are only run post-merge or by ``[testextended][extended:networking]``.

**Comment by marun** [comment]
The fix for this merged with #11613
